docker/standalone-ha.xml

```diff
@@ -101,9 +99,6 @@
             <logger category="org.jboss.as.config">
                 <level name="DEBUG"/>
             </logger>
-            <logger category="org.jgroups">
```

Comment:
yes, we added to check things at infisnispan level. This is a temporal change that we need to evaluate the errors in the clustered mode.